### DISCUSSION TITLE
el-table header 的 scope 问题

### DISCUSSION THREAD

**Discussion by b564115824** [element-plus/element-plus#10843] [Q&A] [answered]
element-ui 中
![image](https://user-images.githubusercontent.com/33363819/205001946-3cfae7d8-78bb-4752-8ff3-fa73fbf2d1cc.png)

如何在 element-plus 中，slot header 使用 scope？
![image](https://user-images.githubusercontent.com/33363819/205002091-cb2b0a1f-ae5c-43da-acb1-55b25957127c.png)

**Comment by ryuhangyeong** [comment, marked as answer]
Is the example below what you are looking for?

https://element-plus.org/en-US/component/table.html#table-column-slots

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtdGFibGUgOmRhdGE9XCJmaWx0ZXJUYWJsZURhdGFcIiBzdHlsZT1cIndpZHRoOiAxMDAlXCI+XG4gICAgPGVsLXRhYmxlLWNvbHVtbiBsYWJlbD1cIkRhdGVcIiBwcm9wPVwiZGF0ZVwiIC8+XG4gICAgPGVsLXRhYmxlLWNvbHVtbiBsYWJlbD1cIk5hbWVcIiBwcm9wPVwibmFtZVwiIC8+XG4gICAgPGVsLXRhYmxlLWNvbHVtbiBhbGlnbj1cInJpZ2h0XCI+XG4gICAgICA8IS0tIGhlcmUgLS0+XG4gICAgICA8dGVtcGxhdGUgI2hlYWRlcj1cInNjb3BlXCI+XG4gICAgICAgIDxoMT5cbiAgICAgICAgICB7eyBzY29wZS4kaW5kZXggfX1cbiAgICAgICAgICB7eyBzY29wZS5jb2x1bW4gfX1cbiAgICAgICAgPC9oMT5cbiAgICAgICAgPGVsLWlucHV0IHYtbW9kZWw9XCJzZWFyY2hcIiBzaXplPVwic21hbGxcIiBwbGFjZWhvbGRlcj1cIlR5cGUgdG8gc2VhcmNoXCIgLz5cbiAgICAgIDwvdGVtcGxhdGU+XG4gICAgICA8dGVtcGxhdGUgI2RlZmF1bHQ9XCJzY29wZVwiPlxuICAgICAgICA8ZWwtYnV0dG9uIHNpemU9XCJzbWFsbFwiIEBjbGljaz1cImhhbmRsZUVkaXQoc2NvcGUuJGluZGV4LCBzY29wZS5yb3cpXCJcbiAgICAgICAgICA+RWRpdDwvZWwtYnV0dG9uXG4gICAgICAgID5cbiAgICAgICAgPGVsLWJ1dHRvblxuICAgICAgICAgIHNpemU9XCJzbWFsbFwiXG4gICAgICAgICAgdHlwZT1cImRhbmdlclwiXG4gICAgICAgICAgQGNsaWNrPVwiaGFuZGxlRGVsZXRlKHNjb3BlLiRpbmRleCwgc2NvcGUucm93KVwiXG4gICAgICAgICAgPkRlbGV0ZTwvZWwtYnV0dG9uXG4gICAgICAgID5cbiAgICAgIDwvdGVtcGxhdGU+XG4gICAgPC9lbC10YWJsZS1jb2x1bW4+XG4gIDwvZWwtdGFibGU+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgY29tcHV0ZWQsIHJlZiB9IGZyb20gJ3Z1ZSdcblxuaW50ZXJmYWNlIFVzZXIge1xuICBkYXRlOiBzdHJpbmdcbiAgbmFtZTogc3RyaW5nXG4gIGFkZHJlc3M6IHN0cmluZ1xufVxuXG5jb25zdCBzZWFyY2ggPSByZWYoJycpXG5jb25zdCBmaWx0ZXJUYWJsZURhdGEgPSBjb21wdXRlZCgoKSA9PlxuICB0YWJsZURhdGEuZmlsdGVyKFxuICAgIChkYXRhKSA9PlxuICAgICAgIXNlYXJjaC52YWx1ZSB8fFxuICAgICAgZGF0YS5uYW1lLnRvTG93ZXJDYXNlKCkuaW5jbHVkZXMoc2VhcmNoLnZhbHVlLnRvTG93ZXJDYXNlKCkpXG4gIClcbilcbmNvbnN0IGhhbmRsZUVkaXQgPSAoaW5kZXg6IG51bWJlciwgcm93OiBVc2VyKSA9PiB7XG4gIGNvbnNvbGUubG9nKGluZGV4LCByb3cpXG59XG5jb25zdCBoYW5kbGVEZWxldGUgPSAoaW5kZXg6IG51bWJlciwgcm93OiBVc2VyKSA9PiB7XG4gIGNvbnNvbGUubG9nKGluZGV4LCByb3cpXG59XG5cbmNvbnN0IHRhYmxlRGF0YTogVXNlcltdID0gW1xuICB7XG4gICAgZGF0ZTogJzIwMTYtMDUtMDMnLFxuICAgIG5hbWU6ICdUb20nLFxuICAgIGFkZHJlc3M6ICdOby4gMTg5LCBHcm92ZSBTdCwgTG9zIEFuZ2VsZXMnLFxuICB9LFxuICB7XG4gICAgZGF0ZTogJzIwMTYtMDUtMDInLFxuICAgIG5hbWU6ICdKb2huJyxcbiAgICBhZGRyZXNzOiAnTm8uIDE4OSwgR3JvdmUgU3QsIExvcyBBbmdlbGVzJyxcbiAgfSxcbiAge1xuICAgIGRhdGU6ICcyMDE2LTA1LTA0JyxcbiAgICBuYW1lOiAnTW9yZ2FuJyxcbiAgICBhZGRyZXNzOiAnTm8uIDE4OSwgR3JvdmUgU3QsIExvcyBBbmdlbGVzJyxcbiAgfSxcbiAge1xuICAgIGRhdGU6ICcyMDE2LTA1LTAxJyxcbiAgICBuYW1lOiAnSmVzc3knLFxuICAgIGFkZHJlc3M6ICdOby4gMTg5LCBHcm92ZSBTdCwgTG9zIEFuZ2VsZXMnLFxuICB9LFxuXVxuPC9zY3JpcHQ+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)

**Reply by b564115824** [reply]
Thanks
